### DISCUSSION TITLE
Filer metadata change subscription

### DISCUSSION THREAD

**Discussion by mbuczko** [seaweedfs/seaweedfs#2413] [General]
Hey, I'm trying to follow this https://github.com/chrislusf/seaweedfs/wiki/Filer-Change-Data-Capture and subscribe to metadata changes:
```sh
weed filer.meta.tail -filer=localhost:8333 -timeAgo=10h
```
and, as expected, it shows a couple of change events. However, every new change (like upload) results in:
```
tail localhost:8333: subscribing filer meta change: rpc error: code = Unavailable desc = error reading from server: EOF
```
Any idea why where the problem with new events comes from?

```
# weed version
version 30GB 2.75 c9d3fb4a3002559502a84228f6d4af16b66a52d1 linux amd6
```

**Comment by chrislusf** [comment]
Can not reproduce. Need exact steps.

**Reply by mbuczko** [reply]
the steps where as easy as calling `weed filer.meta.tail` and uploading file using filer UI. but your comment made me thinking that something could go wrong with my latest upgrade to 2.75. And indeed, after re-instalation it all works perfectly now. I'm not sure what it was, but I'm happy to have seaweed working again :)

thanks.
